From c90bd747e3df80a63da787200a4210d0f23bf9e4 Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Sat, 11 Aug 2018 14:43:26 +0200
Subject: [PATCH 71/74] jni: Resolve -Wwritable-strings warnings in helium FW
 library variables

* fixes: ISO C++11 does not allow conversion from string literal to 'char *'

Change-Id: I76de69d5c2fef3f53c5d57a4fc11b4d8f959ed93
---
 jni/android_hardware_fm.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/jni/android_hardware_fm.cpp b/jni/android_hardware_fm.cpp
index dbdf0b9..9cd8187 100644
--- a/jni/android_hardware_fm.cpp
+++ b/jni/android_hardware_fm.cpp
@@ -82,8 +82,8 @@ static JavaVM *g_jVM = NULL;
 
 namespace android {
 
-char *FM_LIBRARY_NAME = "fm_helium.so";
-char *FM_LIBRARY_SYMBOL_NAME = "FM_HELIUM_LIB_INTERFACE";
+const char *FM_LIBRARY_NAME = "fm_helium.so";
+const char *FM_LIBRARY_SYMBOL_NAME = "FM_HELIUM_LIB_INTERFACE";
 void *lib_handle;
 static int slimbus_flag = 0;
 
-- 
2.17.1

